gcloud/datastore/test_client.py

```diff
@@ -620,8 +620,7 @@ def test_put_multi_no_batch_w_partial_key(self):
         client = self._makeOne(credentials=creds)
         client.connection._commit.append(_CommitResult(key))
 
-        result = client.put_multi([entity])
-        self.assertTrue(result is None)
+        client.put_multi([entity])
```

Comment:
Disabling `assignment-from-no-return` in tests is no skin off my back. Will toss it in and ping you once folded in.